pebblo/app/storage/sqlite_db.py

```diff
@@ -104,6 +111,42 @@ def query_by_id(self, table_obj, id):
             )
             return False, err
 
+    @timeit
+    def query_by_list(
+        self,
+        table_obj: Type[DeclarativeMeta],
+        filter_key: str,
+        filter_values: List[str],
+        max_filter_values: int = 100,
+    ):
+        """
+        Pass filter like list. For example get snippets with ids in [<id1>, <id2>]
+        :param table_obj: Table object on which query is to be performed
+        :param filter_key: Search key
+        :param filter_values: List of strings to be added to filter criteria.
+        :param max_filter_values: Max number ot items to be searched for. Default value is 10.
```

Comment:
The docstring states 10, but the actual value is 100

Reply:
Updated.